spec/myplugin/01-access_spec.lua

```diff
@@ -1,6 +1,7 @@
 local helpers = require "spec.helpers"
 
-describe("Demo-Plugin: myplugin (access)", function()
+for _, strategy in helpers.each_strategy() do
+ describe("Demo-Plugin: myplugin (access)", function()
```

Comment:
The indentation is of. The codebase uses a 2 space indent.

Also: in the describe block please include the strategy name in the description, otherwise it seems it is running all tests twice. See [this line](https://github.com/Kong/kong/blob/master/spec/03-plugins/10-key-auth/02-access_spec.lua#L7) for example.

Reply:
@Tieske pls review!